en/pe-docs/docs/devops/cd-pipelines/submit-and-manage-environment-promotion-approval-requests.md

```diff
@@ -0,0 +1,34 @@
+# Submit and Manage Environment Promotion Approval Requests
+
+In Choreo, when an administrator or platform engineer [configures the Environment Promotion workflow to require approval](../../governance/workflows/configure-approvals-for-choreo-workflows.md), you must submit a request to obtain approval to promote the deployment to a critical environment.
+
+Upon submitting a promotion approval request, Choreo notifies all authorized assignees via email about the [review request](../../governance/approvals/review-workflow-approval-requests.md). When an authorized assignee approves or rejects the request, you will receive an email with details of the decision.
+
+
+## Request approval for environment promotion
+
+### Prerequisites
+
+- You should have a component created, built, and deployed to the first environment configured in your CD pipeline.
+- Ensure that an approval workflow is configured for environment promotion.
+- You should promote the component to a critical environment, such as production.
+- You must have required permissions to promote components or at least `WORKFLOW-MANAGEMENT >> Create approval requests` permission
```

Comment:
```suggestion
- Ensure you have required permissions to either promote the component or to create approval requests.
```

Reply:
we need to be specific about the permission here.

Reply:
Then shall we have something like this:
Ensure you have required permissions to either promote the component or to create approval requests. Approval request creation requires `Create approval requests` permission under `WORKFLOW-MANAGEMENT`.

WDYT?

Reply:
looks good